src/joyride/sci.cljs

```diff
@@ -31,6 +32,28 @@
       {:file ns-path
        :source (str (fs/readFileSync path-to-load))})))
 
+(def ^:private extension-namespace-prefix "ext://")
+
+(defn- extract-extension-name [namespace]
+  (subs namespace (count extension-namespace-prefix)))
+
+(defn- active-extension? [namespace]
+  (when (.startsWith namespace extension-namespace-prefix)
+    (let [[extension-name _module-name] (str/split (extract-extension-name namespace) #"\$")
```

Comment:
I think CLJS also supports a string instead of a regex. Could also just use `.split` perhaps.